<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportProjectDialog</class>
 <widget class="QDialog" name="ExportProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Defines the target architecture.

Gideros Studio supports iOS, Android, Windows Desktop, Mac OS X Desktop and WinRT (Windows Store and Windows Phone apps). Choose your 
target platform to export the application source.</string>
       </property>
       <property name="text">
        <string>Architecture:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="architecture">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Choose your platform here.</string>
       </property>
       <item>
        <property name="text">
         <string>iOS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Android</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Windows Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mac OS X Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WinRT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GApp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Win32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Html5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="architectureTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>iOS</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_51">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <item>
          <widget class="QLabel" name="label1">
           <property name="text">
            <string>Bundle ID:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ios_bundle">
           <property name="text">
            <string>com.yourdomain.yourapp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Android</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_33">
         <item>
          <widget class="QLabel" name="label2">
           <property name="toolTip">
            <string>Defines the package name of the Android application.

Generally speaking, the first 2 package &quot;words&quot; are your
web address in reverse.

Example: com.giderosmobile.timthetimber</string>
           </property>
           <property name="text">
            <string>Select Template:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="android_template">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Eclipse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Android Studio</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_39">
         <item>
          <widget class="QLabel" name="label3_3">
           <property name="toolTip">
            <string>Defines the package name of the Android application.

Generally speaking, the first 2 package &quot;words&quot; are your
web address in reverse.

Example: com.giderosmobile.timthetimber</string>
           </property>
           <property name="text">
            <string>Package Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="packageName">
           <property name="text">
            <string>com.yourdomain.yourapp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>72</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Windows Desktop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_52">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_40">
         <item>
          <widget class="QLabel" name="label4_2">
           <property name="text">
            <string>Organization:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="win_org">
           <property name="text">
            <string>GiderosMobile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label5_2">
           <property name="text">
            <string>Domain:       </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="win_domain">
           <property name="text">
            <string>giderosmobile.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Mac OS X Desktop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_38">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Organization: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="osx_org">
           <property name="text">
            <string>GiderosMobile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_36">
         <item>
          <widget class="QLabel" name="label5">
           <property name="text">
            <string>Domain:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="osx_domain">
           <property name="text">
            <string>giderosmobile.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_37">
         <item>
          <widget class="QLabel" name="label6">
           <property name="text">
            <string>Bundle ID:      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="osx_bundle">
           <property name="text">
            <string>com.yourdomain.yourapp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_32">
         <item>
          <widget class="QLabel" name="label3_2">
           <property name="text">
            <string>App Category:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="osx_category">
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>WinRT</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_61">
       <item>
        <widget class="QLabel" name="label_61">
         <property name="text">
          <string>
Use this to create Windows Phone and Windows Store apps (&quot;Metro Style&quot; apps)
A universal solution is created containing a Windows Phone and Windows Store project.
</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_35">
         <item>
          <widget class="QLabel" name="label4">
           <property name="text">
            <string>Package Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="winrt_package">
           <property name="text">
            <string>com.yourdomain.yourapp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_34">
         <item>
          <widget class="QLabel" name="label3">
           <property name="text">
            <string>Organization:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="winrt_org">
           <property name="text">
            <string>GiderosMobile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>GApp</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>(ALPHA) Generate a single .GApp file playable by QT-Based desktop players.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1_2">
      <attribute name="title">
       <string>Win32</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>
Beta version. Should be feature complete but use with caution!

This is the same as Windows Desktop export except it produces a lightweight export without Qt libraries. The package is smaller and may run smoother. 

The exported program will attempt to use VSYNC to time the drawing of frames (30 or 60 fps, whichever you have selected). If VSYNC (which is graphics card dependent) is not detected on the PC, it will fall back to Windows timer events. To overide this behaviour, and ALWAYS uses timer events, launch with &quot;win32 timer&quot;

win32.exe and win32console.exe in export are identical except the latter runs without a console window popping up.

* No extra settings for Win32.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer12_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Html5</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Export for the web. Generates a set of html and javascript files suitable for a WebGL compatible browser. You can restrict your app to be launched from a specific hostname by entering it below (i.e. www.example.com). Leave the field empty to allow it to be run from any host. When this field is not empty, encryption is automatically turned on.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Host name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="html5_host"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Maximum memory usage (in MB), adjust according to your app needs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="html5_mem"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="exportMode">
       <property name="currentText">
        <string>Full export</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Full export</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Assets only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Player (no assets)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="encryptAssets">
       <property name="text">
        <string>Encrypt asset files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="encryptCode">
       <property name="text">
        <string>Encrypt code files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plugins_choose">
       <property name="text">
        <string>Select plugins...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
